identifiers/country-us/state-ar-local_gov.csv

```diff
@@ -64,9 +66,20 @@ ocd-division/country:us/state:ar/county:carroll/council_district:6,Carroll Count
 ocd-division/country:us/state:ar/county:carroll/council_district:7,Carroll County District 7
 ocd-division/country:us/state:ar/county:carroll/council_district:8,Carroll County District 8
 ocd-division/country:us/state:ar/county:carroll/council_district:9,Carroll County District 9
-ocd-division/country:us/state:ar/county:carroll/council_district:10,Carroll County District 10
-ocd-division/country:us/state:ar/county:carroll/council_district:11,Carroll County District 11
+ocd-division/country:us/state:ar/county:clark/council_district:1,Clark County Justice of the Peace District 1
```

Comment:
I think we should try to keep the names of the Justice of the Peace districts consistent in the file.

So either all of them should be `<County name> Justice of the Peace District <district number>` or `<County name> District <district number>`

Reply:
ok, I reverted the district names to follow the prior precedent of `<County name> District <district number>`